src/HeadTag.js

```diff
@@ -50,3 +39,28 @@ export default class HeadTag extends Component {
     return null;
   }
 }
+// eslint-disable-next-line no-unused-expressions
+process.env.NODE_ENV !== 'production'
+  ? (HeadTag.contextTypes = {
+      reactHeadTags: PropTypes.object,
+    })
+  : undefined;
```

Comment:
Would it be possible to use something like [babel-plugin-transform-react-remove-prop-types](https://www.npmjs.com/package/babel-plugin-transform-react-remove-prop-types) instead of defining it like this?